samples/sensor/mcux_lpcmp/boards/frdm_mcxn947_mcxn947_cpu0.overlay

```diff
@@ -0,0 +1,9 @@
+/*
+ * Copyright 2024 NXP
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+&lpcmp0 {
+    function-clock = <3>;
```

Comment:
would be nice to have either a string enum or an include header in include/zephyr/dt-bindings to clarify the meaning of these magic numbers